.github/workflows/test.yml

```diff
@@ -44,7 +59,8 @@ jobs:
     - name: Test with pytest (macOS)
       if: runner.os == 'macOS'
       run: |
-        pytest --cov=vorta
+        pytest
```

Comment:
Is there a need to remove code coverage from macOS?

Reply:
Yes, pytest+coverage would hang indefinitely after successfully completing tests. Looks like you can't [reliably have](https://github.com/pytest-dev/pytest-cov#overview) coverage + subprocesses (which we have). Using the pytest-cov plugin didn't help either.

Examples of "hung" tests:

- https://github.com/borgbase/vorta/actions/runs/587040055
- https://github.com/borgbase/vorta/actions/runs/585924308
- https://github.com/borgbase/vorta/actions/runs/585837648